src/PropertyTransformHelper.js

```diff
@@ -63,26 +63,32 @@ const doCompactPropertyLiteralMapping = (messageFieldsMapping, msgName, pkgName,
     if (compacts.remaining() == 0){
         return '';
     }
-    let tagNumber = compacts.readUint32();
```

Comment:
Why did you change to uint64? Protobuf tags should be only 32 bytes. Is the go implementation faulty?

Reply:
I just realized that it indeed is. This is due to us using the same type for slice indexes and tag names. We should fix that in Go and only use uint64 for slice length. Would you like to fix that if I increase the bounty on this a bit more? It would be great to get this merged in at the same time. If not though it's ok to commit it like that.

Reply:
@lucasvo change is done,  actually I also make change for GO side on my forked repo, just do not know vedhavyas have done the work.

Reply:
vedhavyas's PR is for another problem, have submitted a PR on Go side